app/controllers/messages_controller.rb

```diff
@@ -146,12 +146,12 @@ def quote
     content = "#{ll(Setting.default_language, :text_user_wrote, user)}\\n> "
     content << text.to_s.strip.gsub(%r{<pre>((.|\s)*?)</pre>}m, '[...]').gsub('"', '\"').gsub(/(\r?\n|\r\n?)/, '\\n> ') + '\\n\\n'
     render(:update) { |page|
-      page << "$('message_subject').value = \"#{subject}\";"
-      page.<< "$('message_content').value = \"#{content}\";"
+      page << "$('reply_subject').value = \"#{subject}\";"
+      page.<< "$('reply_content').value = \"#{content}\";"
       page.show 'reply'
```

Comment:
I'd suggest we use every opportunity we can get to rip out Prototype, even if out-of-scope (arguably). See fda43ac3045dd18cadcfb331e266373f572fc742 as an example.